### DISCUSSION TITLE
[HELP] Install Diablo on MAC with devilutionX

### DISCUSSION THREAD

**Discussion by AleMoschi** [diasurgical/DevilutionX#7358] [Q&A] [answered]
Hi everyone, I need help installing Diablo on Mac with DevilutionX.

So, I download and place the app in the Mac Applications folder. I launch the program, and it tells me that the file "DIABDAT.MPQ" does not exist.

So, as per the instructions, I go to "/Library/Application Support/", but I can't find the "diasurgical" folder. So, I create it, and inside that folder, I create another folder called "devilution" and place the files "DIABDAT.MPQ" and "devilutionx.mpq" in there.

But it still doesn't work. Where am I going wrong?

**Comment by AJenbo** [comment, marked as answer]
It says `~/Library/Application Support` (user folder) not `/Library/Application Support/` (system folder)

**Reply by AleMoschi** [reply]
Oh, thank you!
